h/static/scripts/group-forms/components/test/AnnotationCard-test.js

```diff
@@ -0,0 +1,138 @@
+import { mockImportedComponents, mount } from '@hypothesis/frontend-testing';
+
+import { Config } from '../../config';
+import AnnotationCard, { $imports } from '../AnnotationCard';
+
+describe('AnnotationCard', () => {
+  let fakeConfig;
+  let fakeAnnotation;
+  let fakeQuote;
+  let fakeUsername;
+
+  beforeEach(() => {
+    fakeConfig = {
+      context: {},
+    };
+    fakeAnnotation = {
+      tags: [],
+      links: {},
+    };
+
+    fakeQuote = sinon.stub().callsFake(anno => anno.quote);
+    fakeUsername = sinon.stub();
+
+    $imports.$mock(mockImportedComponents());
+    $imports.$mock({
+      '../utils/annotation-metadata': {
+        username: fakeUsername,
+        quote: fakeQuote,
+      },
+      '@hypothesis/annotation-ui': {
+        AnnotationTimestamps: () => null,
+        AnnotationUser: () => null,
+        AnnotationGroupInfo: () => null,
+        MarkdownView: () => null,
```

Comment:
This works for the moment, but I think it would make sense to add an option to `mockImportedComponents` to opt-in to mocking certain third-party components. One reason is that `mockImportedComponents` gives its mocks useful behaviors like rendering children.